starter-code/views/login.ejs

```diff
@@ -0,0 +1,25 @@
+<link rel='stylesheet' href='/stylesheets/style.css' />
+
+<h2>Login Page</h2>
+
+<form action="/login" method="POST">
+  <label>
+    Username:
+  </label>
+  <input type="text" name="username">
```

Comment:
Did you know you can put the `input` tag *inside* the `label` tag? Like this:

```html
<label>Username: <input type="text" name="username"></label>
```

The difference? Try to click on the text *Username* now.

Reply:
Good tip 👍 (for Nico: it selects the field)